Makefile

```diff
@@ -82,6 +87,12 @@ ifeq ($(TARGET),arm)
 #CC := arm-rdk-linux-gnueabi-gcc -mthumb -mfpu=vfp -mcpu=cortex-a9 -mfloat-abi=soft -mabi=aapcs-linux -mno-thumb-interwork -ffixed-r8 -fomit-frame-pointer
 # CFLAGS will be overriden by Caller as required
 INC_DIRS += $(UT_DIR)/sysroot/usr/include
+ifndef SDKTARGETSYSROOT
```

Comment:
our docker shouldn't have the SSL libraries or openssl, so we shouldn't use it.

you can't assume that it's available for arm and that sysroot is set, you have to build it.